content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -395,6 +395,19 @@ Used on: Service
 
 The control plane adds this label to an Endpoints object when the owning Service is headless.
 
+### service.kubernetes.io/topology-aware-hints {#servicekubernetesiotopology-aware-hints}
+
+Example: `service.kubernetes.io/topology-aware-hints: "auto"`
+
+Used on: Service
+
+This annotation is used for enabling [Topology Aware Hints](/docs/concepts/services-networking/topology-aware-hints/) for a Service.
```

Comment:
```suggestion
This annotation was used for enabling _topology aware hints_ on Services. Topology aware
hints have since been renamed: the concept is now called
[topology aware routing](/docs/concepts/services-networking/topology-aware-routing/).
```